src/Xunit.Combinatorial.Tests/CombinatorialRangeAttributeTests.cs

```diff
@@ -0,0 +1,46 @@
+namespace Xunit.Combinatorial.Tests
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Linq;
+    using System.Text;
+    using System.Threading.Tasks;
+
+    public class CombinatorialRangeAttributeTests
```

Comment:
These are good tests. Can you add tests that actually *use* `CombinatorialRangeAttribute`? While the test body itself may not be able to *assert* much (aside from the arg being _one of_ the expected values), it would be good to see it demonstrated and verify visually that Test Explorer or the test log actually invoked the test that many times.

Reply:
hint: I think you'll find the new attribute doesn't work as-is. I think you'll need to change `ValuesUtilities.cs` to recognize the added attribute type.

Reply:
Added another local variable to `GetValuesFor()` to cover both Combinatorial attributes. I recognize that this solution is not good for adding more attributes in the future but a better course of action did not come to me easily, so I hope this is okay.

The test explorer is showing all 10 cases from the test I added to `SampleUses.cs` and they're passing with some bare-bones assertions.